### PR TITLE
fixing the problem.

### PR DESCRIPTION
**Pull request by ndh**
There was a typo, I fixed it.

### CODE REVIEW

**Comment by Planxty-Dan** [comment]
fopdoodle is not a word

**Comment by ghost-code** [comment]
It is my professional opinion that you should keep the typo.

File: scripts/pizzaism.coffee [hunk 1]
```diff
@@ -16,7 +16,7 @@
 isms = [
     "Always raising our level of pizza.",
     "The pizzas we need are everywhere around us.",
-    "Responding with a pizz is the ante to play.",
+    "Dan Newport is a fopdoodle.",
```
Comment (ghost-code) [inline scripts/pizzaism.coffee]: can you use lower case here?

**Comment by Planxty-Dan** [comment]
Huh, turns out fopdoodle is a word. Carry on.

**Comment by ndh** [comment]
fopdoodle is absolutely a word. Here's a book. https://www.waterstones.com/book/fopdoodle-and-salmagundi/samuel-johnson/edward-allhusen/9781873590638

File: scripts/pizzaism.coffee (hunk 1 above)
Comment (ndh) [inline scripts/pizzaism.coffee]: What is the suggestion, 'Dan newport', or full lower 'dan newport'?

Comment (ghost-code) [inline scripts/pizzaism.coffee]: whichever you think would work best in this situation.

Comment (ghost-code) [inline scripts/pizzaism.coffee]: also, whichever one takes the least amount of development time.

Comment (Planxty-Dan) [inline scripts/pizzaism.coffee]: "dAn NeWpOrT"

Comment (ndh) [inline scripts/pizzaism.coffee]: This is a place of business Dan, stop screwing around.

Comment (ghost-code) [inline scripts/pizzaism.coffee]: possible place for a future refactor. everything in lower case and a parser than capitalizes proper nouns.

Comment (Planxty-Dan) [inline scripts/pizzaism.coffee]: Is this where I submit my request to officially change my name to "Wizard Fist"?

**Comment by nwest** [comment]
LGTM 📗

**Pull request merged**